src/spikeinterface/qualitymetrics/tests/conftest.py

```diff
@@ -0,0 +1,37 @@
+import pytest
```

Comment:
Added this file, to share fixtures across test files. See: https://docs.pytest.org/en/6.2.x/fixture.html#conftest-py-sharing-fixtures-across-multiple-files